docs/reqstool/reqstool_config.yml

```diff
@@ -1,6 +1,6 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/Luftfartsverket/reqstool-client/main/src/reqstool/resources/schemas/v1/reqstool_config.schema.json
 
-type: default
```

Comment:
Why this change? I've updated the docs regarding reqstool_config's new format in the docs on my branch.

Reply:
Reverted, have you also fixed the Requirements annotation? If so we can just close this PR.